src/quoridor/game/position.go

```diff
@@ -26,3 +26,44 @@ func (p Position) Copy(deltaColumn int, deltaRow int) Position {
 	copy = copy.translateRow(deltaRow)
 	return copy
 }
+
+type Positions []Position
+
+func (positions Positions) IndexOf(e Position) int {
```

Comment:
You can maybe use generics for all our indexOf functions?

Reply:
I do not know how to achieve that goal because I use a dedicated method inside this method -> Equals which is not generic

Reply:
It seems that you can use interfaces for that

https://dlintw.github.io/gobyexample/public/interfaces.html

Reply:
hard to use it with an array